hr_employee_id/views/hr_employee_views.xml

```diff
@@ -45,9 +45,11 @@
         <field name="inherit_id" ref="hr.hr_kanban_view_employees"/>
         <field name="arch" type="xml">
             <li id="last_login" position="after">
-                <li t-if="record.identification_id.raw_value">
-                    <field name="identification_id"/>
-                </li>
+                <t t-if="record.identification_id">
+                    <li t-if="record.identification_id.raw_value" >
+                        <field name="identification_id" groups="hr.group_hr_user"/>
```

Comment:
It's not better to add the `groups` option 2 lines above, into the `t t-if` tag?

Reply:
Hum, it's indeed better, but I wanted to keep it as close as possible to the standard, as another module may want to update this field and it could lead to broken things

Reply:
I just wonder if, for users non-belonging to the given group, this would not display as a dangling dash due to an empty `<li>` tag.